### PR TITLE
refactor: redis type definitions

### PR DESCRIPTION
**Pull request by kuyh**
http://git.sphd.io/support/stt-issues/issues/1958

### CODE REVIEW

File: index.d.ts [hunk 1]
```diff
@@ -317,7 +317,7 @@ declare namespace redis {
         hexistsAsync(...args:any[]): Promise<string>
         incrby(args:any[], callback?:redis.ResCallbackT<any>): boolean;
         incrby(...args:any[]): boolean;
-        incrbyAsync(...args:any[]): Promise<string>
+        incrbyAsync(...args:any[]): Promise<numbser>
```
Comment (heycalmdown) [inline index.d.ts]: uh-oh!

Comment (0Ams) [reply index.d.ts]: typo!

File: index.d.ts [hunk 2]
```diff
@@ -231,16 +231,16 @@ declare namespace redis {
         smembersAsync(...args:any[]): Promise<string[]>
         zadd(args:any[], callback?:redis.ResCallbackT<any>): boolean;
         zadd(...args:any[]): boolean;
-        zaddAsync(...args:any[]): Promise<string>
+        zaddAsync(...args:any[]): Promise<number>
         zincrby(args:any[], callback?:redis.ResCallbackT<any>): boolean;
         zincrby(...args:any[]): boolean;
-        zincrbyAsync(...args:any[]): Promise<string>
+        zincrbyAsync(...args:any[]): Promise<any>
```
Comment (sshtel) [inline index.d.ts:237]: Some refactoring is changed to `number` but some are `any` or `number | undefined`.
is there any reason for this?

Comment (kuyh) [reply index.d.ts:237]: because redis command returns that type - https://redis.io/commands

Comment (sshtel) [reply index.d.ts:237]: Ok, for zincrby command, `any` makes sense.

**Review by sshtel — APPROVED** [review]

**Review by belldoor — APPROVED** [review]

**Pull request merged**